jobs/JEVS_MESOSCALE_PLOTS

```diff
@@ -52,9 +52,9 @@ export SENDMAIL=${SENDMAIL:-NO}
 ####################################
 # Run setpdy and initialize PDY variables
 ####################################
-export cycle=${cycle:-t${cyc}z}
-export cyc=${cyc:-00}
-export cycle=t${cyc}z
+export cycle=${cycle:-t${vhr}z}
+export vhr=${vhr:-00}
+export cycle=t${vhr}z
```

Comment:
@BinbinZhou-NOAA , In JEVS_MESOSCALE_PLOTS, I think line 57 is redundant. Line 55 is all that is needed.

Reply:
@ShelleyMelchior-NOAA But shouldn't `export vhr=${vhr:-00}` come before `export cycle=${cycle:-t${vhr}z}`?